runner/execer/os/os_execer.go

```diff
@@ -249,64 +249,82 @@ type osProcGetter struct{}
 // Given this list of pids, find all processes with a pgid or ppid in that set, and modify the set in place.
```

Comment:
Comment out of date - would just describe what the function does, but leave how it does it to inline comments